config.properties

```diff
@@ -0,0 +1,3 @@
+db.url=jdbc:postgresql://localhost:5432/assignment-3
+db.username=braedenkloke
```

Comment:
this should probably be non-initiated in the repo and instead be up to each local definition (with a note in the readme)